components/server/src/workspace/gitpod-server-impl.ts

```diff
@@ -3464,14 +3463,12 @@ export class GitpodServerImpl implements GitpodServerWithTracing, Disposable {
     //
     //#endregion
 
+    /**
+     * TODO(ak)
+     * @deprecated remove it after dashboard is deployed. It was replaced with error reporting in GCP.
+     */
     async reportErrorBoundary(ctx: TraceContextWithSpan, url: string, message: string): Promise<void> {
```

Comment:
Not sure maybe it is fine to delete it, there can be some old dashboards immediately after deployment. So can be another PR. cc @geropl wdyt?

Reply:
Although this is not important specifically, I'd love us to do more incremental rollouts. :+1: